exec/cnex/util.h

```diff
@@ -1,8 +1,8 @@
 #ifndef UTIL_H
 #define UTIL_H
 
-#define estr(x)     #x
-#define ENUM(x)     estr(x)
+#define estr(x)                 #x
```

Comment:
Why are there whitespace changes here? There should be no change if you're not changing anything.

Reply:
This was only because of the last define which was removed.  This file will be restored from HEAD.